pkg/utils.go

```diff
@@ -67,24 +67,50 @@ func (duration *Duration) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+//	func Convert(i interface{}) interface{} {
```

Comment:
Do we still need this commented out code? If so can you add a comment what it might be needed for?